rackhd_BIST/test_suites.py

```diff
@@ -537,35 +539,44 @@ def get_supported_skus(self):
         """
         http_connect = self.__initiate_rackhd_connect()
         http_connect.request(self.http_method, self.get_sku_api)
```

Comment:
A general suggestion, use try-catch to pack http_connect.request as a method instead of use try-catch for each method. In this way we can use it if in the future if we want to expand API tests.